src/main/kotlin/day17/dto/MediaDto.kt

```
@@ -0,0 +1,60 @@
+    @SerialName("created_at") val createdAt: String? = null
+)
+
+object FallbackSerializer : KSerializer<Int> {
+    override val descriptor: SerialDescriptor = PrimitiveSerialDescriptor("MediaDto", PrimitiveKind.STRING)
+
+    override fun deserialize(decoder: Decoder): Int {
+        check(decoder is JsonDecoder)
+        return decoder.decodeJsonElement().jsonPrimitive.content.toInt()
+    }
+
+    override fun serialize(encoder: Encoder, value: Int) {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)